content/en/docs/setup/independent/high-availability.md

```diff
@@ -266,81 +266,87 @@ Please select one of the tabs to see installation instructions for the respectiv
 
 {{% /tab %}}
 {{% tab name="Static Pods" %}}
-**Note**: This is only supported on nodes that have the all dependencies for the kubelet installed. If you are hosting etcd on the master nodes, this has already been set up. If you are hosting etcd on dedicated nodes, you should either use systemd or run the [installation guide](/docs/setup/independent/install-kubeadm/) on each dedicated etcd machine.
+**Note**: This is only supported on nodes that have the all dependencies for the
+kubelet installed. If you are hosting etcd on the master nodes, this has already
+been set up. If you are hosting etcd on dedicated nodes, you should either use
+systemd or run the [installation guide](/docs/setup/independent/install-kubeadm/)
+on each dedicated etcd machine.
 
 Run the following to generate the manifest file:
 
-<!-- Using indentation instead of code fencing because of https://github.com/russross/blackfriday/issues/239 -->
```

Comment:
Given that there's no numbering in this section, it wasn't clear to me why we needed to go the indentation route, knowing the detrimental impact it has on copypaste-ability as discussed in https://github.com/kubernetes/website/pull/8663.